src/streams/StreamScreen.js

```diff
@@ -59,29 +60,29 @@ class StreamScreen extends PureComponent<Props> {
     const { styles } = this.context;
 
     return (
-      <Screen title="Stream" padding>
+      <Screen title="Stream">
         <StreamCard stream={stream} subscription={subscription} />
-        <OptionDivider />
         <OptionRow
           label="Pinned"
           defaultValue={subscription.pin_to_top}
           onValueChange={this.handleTogglePinStream}
         />
-        <OptionDivider />
         <OptionRow
           label="Muted"
           defaultValue={stream.in_home_view === false}
           onValueChange={this.handleToggleMuteStream}
         />
-        <OptionDivider />
         <OptionRow
           label="Notifications"
           defaultValue={subscription.push_notifications}
           onValueChange={this.toggleStreamPushNotification}
           customStyle={this.context.styles.backgroundColor}
         />
-        <ZulipButton style={styles.marginTop} text="Topics" onPress={this.handleTopics} />
-        <ZulipButton style={styles.marginTop} text="Edit" onPress={this.handleEdit} />
+        <OptionDivider />
+        <View style={styles.screenPadding}>
+          <ZulipButton text="Topics" onPress={this.handleTopics} />
+          <ZulipButton style={styles.marginTop} text="Edit" onPress={this.handleEdit} />
```

Comment:
Hmm, I'm still not a fan of the `marginTop` on this button -- it feels like a pretty ad-hoc way of handling the layout. But that's how lots of other ZulipButtons around the app work, so (unlike with `StreamCard`) we can't just push the padding inside the component without fixing up lots of other use sites to match. So fixing that is beyond the scope of this PR.